### ISSUE TITLE
Add documentation about interactions with RadioGroup 

### ISSUE THREAD

**Issue by MonsieurNiark** [mvysny/karibu-testing#107]
Hello,

I am struggling while trying to interact with a dynamic radio group, and I do not find any example in code or documentation (except something about getItemsLabel, which is returning me "Div[]" x2.

 ```
       final RadioButtonGroup<String> radioButtonGroup = new RadioButtonGroup<>();
        radioButtonGroup.setItems(userOrganisations.stream().map(OrganisationDto::getLibelle).collect(Collectors.toList()));
        radioButtonGroup.setRenderer(new ComponentRenderer<>(nomOrga -> new Div(new Paragraph(nomOrga))));
        radioButtonGroup.setId(ConstantesChoixOrganisationID.FCHOIXORGANISATION_RADIOBUTTON_ORGANISATION);
        radioButtonGroup.setClassName("choix-organisation-radiobouton");
```

They are rendering in div, because my radio group acting like a choice menu. 
Any idea on how to interact with them?

Thank you.

**Comment by mvysny** [comment]
Actually the code is very simple:

```kotlin
public fun <T : Any> RadioButtonGroup<T>._getRenderedComponent(item: T): Component =
    itemRenderer.createComponent(item)
```

**Comment by mvysny** [comment]
Implemented in Karibu-Testing 1.3.13
